### DISCUSSION TITLE
Some way to manage the other npm alternatives (pnpm/yarn) since they have their own globally installed packages

### DISCUSSION THREAD

**Discussion by josh-hemphill** [Devolutions/UniGetUI#2684] [General]
There's also the issue of updating npm/pnpm/yarn themselves. I've been updating pnpm via winget but that keeps breaking and I think they've released their own self-updater that manages another process to keep the update from breaking itself on windows. Maybe Node's corepack would be the answer since it can manage installing/updating any of the three?

But for me having the NPM update management is useless since my default for everything is PNPM, so some way to change the default manager binary or the ability to manage updates to all three's global packages would be useful.

**Comment by marticliment** [comment]
I will investigate this

**Comment by JaneX8** [comment]
This is exactly what I'm interested in too. I ditched npm entirely and reinstalled all globally installed Node.js packages using Yarn. 

I would really love to see Yarn support in UniGetUI.

**Comment by josh-hemphill** [comment]
Just saw there's a feature request opened for this: #3506
